components/kyma-environment-broker/internal/broker/instance_deprovision.go

```diff
@@ -95,9 +107,9 @@ func (b *DeprovisionEndpoint) Deprovision(ctx context.Context, instanceID string
 	}, nil
 }
 
-func isOngoingNotTemporaryDeprovisioning(existingOperation *internal.DeprovisioningOperation) bool {
-	return !(existingOperation == nil ||
-		existingOperation.Temporary ||
-		existingOperation.State == domain.Failed ||
-		existingOperation.State == domain.Succeeded)
+func isSucceededNotTemporaryButWithNoSteps(existingOperation *internal.DeprovisioningOperation) bool {
```

Comment:
nit: the name is quite verbose, how would you feel about something shorter, maybe something along the lines of -
```suggestion
func retriggerable(existingOperation *internal.DeprovisioningOperation) bool {
```

Reply:
Removed